src/test/resources/features/Scoring/Student_Question/StudentQuestion.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Student Question
+
+  Background:
+    Given user prepare auth request
+    When user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user prepare quiz request with batchCode "futur3"
+    And user hit get all quiz endpoint
+    And user get first quiz id and store id
+    And user get first quiz and store id
+    And user prepare student question request with batchCode "futur3"
```

Comment:
I found it best to always call `And user hit logout endpoint` in each `Background`, since previous tests may alter the state of `authData` singleton